amazon-s3/7.0/modules/ROOT/pages/amazon-s3-connector-reference.adoc

```diff
@@ -2852,10 +2852,10 @@ Configures the bucket website response type.
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Error Document Key a| String | Error document key. |  |
-| Index Document Suffix a| String | Index document suffix. |  |
-| Redirect All Requests To a| <<RedirectRule>> | Redirect all requests to. |  |
-| Routing Rules a| Array of <<RoutingRule>> | Routing rules. |  |
+| Error Document Key a| String | Specifies the key of the error document that Amazon S3 returns when a request results in an error. |  |
+| Index Document Suffix a| String | Defines the suffix that Amazon S3 adds to requests for the bucket root or any subdirectories. |  |
+| Redirect All Requests To a| <<RedirectRule>> | Specifies a rule to redirect all requests. |  |
+| Routing Rules a| Array of <<RoutingRule>> | Array of rules that define redirects based on request criteria. |  |
```

Comment:
```suggestion
| Routing Rules a| Array of <<RoutingRule>> | Defines an array of rules that specify redirects based on request criteria. |  |
```
Changing the sentence to star with a verb in present simple to match the other entries.